### PR TITLE
Create `ACCESS_LIST.md`

### PR DESCRIPTION
**Pull request by confused-Techie**
This PR is a followup to a discussion on Discord. About creating and maintaining a resource that can helpfully point all of us in the right direction of knowing who has access to what, and who doesn't have access to what.

Allowing us to coordinate better and avoid any pitfalls of not following least access privilege's, or having to few members with access to other resources.

### CODE REVIEW

**Comment by Daeraxa** [comment]
This is an awesome start, thanks for putting this together

File: ACCESS_LIST.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+## Google Search
+
+* [@confused-Techie]
+
+## Vultr
+
+* Owner: [@kaosine]
+* Member: [@confused-Techie]
+
+## Coveralls
```
Comment (Meadowsys) [inline ACCESS_LIST.md:133]: the fact that there isn't a newline after Coveralls lowkey bothers me >.>

Comment (confused-Techie) [reply ACCESS_LIST.md]: Meant to only include `Codecov` as I had them confused. Removed `Coveralls`

**Comment by confused-Techie** [comment]
Alright as discussed on Discord, I'll move this to a `.github-private` repo rather than here. Thanks everyone for the reviews.

**Pull request closed without merging**